src/patternfly/components/Button/styles.scss

```diff
@@ -13,6 +13,8 @@
   --pf-c-button--BoxShadowSpread: var(--pf-global--BorderWidth--sm);
   --pf-c-button--Color: var(--pf-global--primary-color);
   --pf-c-button--BorderRadius: var(--pf-global--BorderRadius);
+  --pf-c-button--BorderColor--transparent: var(--pf-global--BorderColor--Transparent);
+  --pf-c-button--BorderWidth: var(--pf-global--BorderWidth--sm);
```

Comment:
Good catch on me using difference variables...too much copy/paste.

My thought with `--BorderColor--transparent` was to set things up with a bit of future-proofing in the event that we'd have different default border colors. If that isn't a concern, your suggestion to change them all to `--BorderColor: var(--pf-global--BorderColor--Transparent);`.

Reply:
Yeap, what Jenn said.

I'd hardcode value `transprarent` and not use a global variable since in this case is not something you'd want to change but a hack to replace borders.

In my opinion, if in the future we want to change it, the effort to remove the border shadow and replace the transparent for a variable should be the same either with a global transparent variable or not.